tests/backup/backup_resiliency_test.go

```diff
@@ -493,12 +492,8 @@ var _ = Describe("{RestartBackupPodDuringBackupSharing}", func() {
 			dash.VerifyFatal(err, nil, "Getting px-backup namespace")
 			err = DeletePodWithLabelInNamespace(pxbNamespace, backupPodLabel)
 			dash.VerifyFatal(err, nil, "Restart backup pod when backup sharing is in-progress")
-			pods, err := core.Instance().GetPods("px-backup", backupPodLabel)
-			dash.VerifyFatal(err, nil, "Getting px-backup pod")
-			for _, pod := range pods.Items {
-				err = core.Instance().ValidatePod(&pod, 5*time.Minute, 30*time.Second)
-				log.FailOnError(err, fmt.Sprintf("Failed to validate pod [%s]", pod.GetName()))
-			}
+			err = ValidatePodByLabel(backupPodLabel, pxbNamespace, 5*time.Minute, 30*time.Second)
+			log.FailOnError(err, "Checking if px-backup pod is in running state")
```

Comment:
`log.FailOnError` won't let us know status when everything is fine. `dash.VerifyFatal` will also print if successful.

Reply:
I do not want the logs to be printed if everything is fine. This is just an intermittent issue. So this is not the main focus of the test, hence I have not used `dash.VerifyFatal`